aiida_orca/parsers/__init__.py

```diff
@@ -40,9 +37,12 @@ def parse(self, **kwargs):
         if fname_out not in out_folder._repository.list_object_names():  #pylint: disable=protected-access
             raise OutputParsingError('Orca output file not retrieved')
 
-        parsed_obj = io.ccread(os.path.join(out_folder._repository._get_base_folder().abspath, fname_out))  #pylint: disable=protected-access
-
-        parsed_dict = parsed_obj.getattributes()
+        try:
+            with self.retrieved.open(fname_out) as handler:
+                parsed_obj = ccread(handler.name)
+                parsed_dict = parsed_obj.getattributes()
+        except:  #pylint: disable=bare-except
+            parsed_dict = None
```

Comment:
I think at this point we should throw a ValueError exception with some error message or something, or ideally return an custom exit code via aiida, but that can be done later. If we don't throw here, we'll get an exception below anyway.